src/components/SearchBar/SearchBar.component.tsx

```diff
@@ -1,22 +1,66 @@
+import { useState } from 'react';
+
+import { topProductData } from 'data/TopProduct.data';
+import { useNavigate } from 'react-router-dom';
+
 import SearchIcon from '@mui/icons-material/Search';
-import { Box, Input } from '@mui/material';
+import { Autocomplete, Box, Stack, TextField } from '@mui/material';
+
+import { useDebounce } from '@hooks/useDebounce';
 
 import { useStyles } from './SearchBar.styles';
 
 export const SearchBar = () => {
     // Styles
     const { classes } = useStyles();
 
+    const navigate = useNavigate();
+
+    const [search, setSearch] = useState('');
+
+    const handleSearch = (event: React.KeyboardEvent) => {
+        if (event.key === 'Enter' && search.trim()) {
+            const formatted = search.trim().toLowerCase().replace(' ', '-');
```

Comment:
```suggestion
            const formattedSearchText = search.trim().toLowerCase().replace(' ', '-');
```

please use descriptive variable names